tools/http3-test/tests/httpbin_tests.rs

```diff
@@ -0,0 +1,692 @@
+#[cfg(test)]
+mod httpbin_tests {
```

Comment:
I think you can remove the two lines above, as files under `tests/` should only be built when testing (i.e. with  `cargo test`), and then you can remove the indentation for the rest of the file.

Reply:
the nice property of them being in `mod httpbin_tests` was that we could make different kinds of integration test and run them separately e.g. `cargo test httpbin_tests::drip` and `cargo test some_other_test::break_it`.

Is there some other way to achieve that ?

Reply:
I guess not, let's just leave as-is then.